### ISSUE TITLE
Not possible to pass options along to MongoConnection when using RemoteCollectionDriver

### ISSUE THREAD

**Issue by Broham** [meteor/meteor#6934]
When using the `RemoteCollectionDriver` you can pass along an `options` object after the connection string.  But from what I can tell, these options are not passed along when making the `MongoConnection`

The code for [creating the RemoteCollectionDriver](https://github.com/meteor/meteor/blob/abd574f38008b45f5e2a6bc322b10bcdde44763a/packages/mongo/remote_collection_driver.js#L4):

```
MongoInternals.RemoteCollectionDriver = function (
  mongo_url, options) {
  var self = this;
  self.mongo = new MongoConnection(mongo_url, options); //Options passed along here
};
```

When the MongoConnection constructor is called the options are passed along, _but_ as far [as I can tell](https://github.com/meteor/meteor/blob/abd574f38008b45f5e2a6bc322b10bcdde44763a/packages/mongo/mongo_driver.js#L156) the only option that is actually used is `poolSize`.  The comments in the code seem to back this theory up:

```
  // XXX maybe we should have a better way of allowing users to configure the
  // underlying Mongo driver
  if (_.has(options, 'poolSize')) {
```

Would it be worth while to update the code to include all of the options that are passed to the `MongoConnection` constructor rather than just using `poolSize`?  In my particular situation I need to connect directly to a hidden secondary MongoDB instance and I have no way to specify that connecting to a secondary without a primary is ok.

**Comment by abernix** [comment]
see #6958 (newly created).  This should probably be looked at during the upgrade of the Mongo driver.  Perhaps close this @Broham, if you think it'll be covered by that issue. 😄

**Comment by abernix** [comment]
@Broham This is a great idea, but I'm closing this because it will be addressed by #6958.  Please bring any additional conversation you might have to that issue!  If you believe I'm wrong, please feel free to post back with additional information and @mention me and I'll reopen.

**Comment by abernix** [comment]
I'll also point out that the other issue is labeled with `pull-requests-encouraged` so if you want to discuss and build, it would a most welcome contribution to the Meteor project and likely to be accepted to core.
